tests/integration/testlib/cmdlib.py

```diff
@@ -25,7 +25,7 @@
 RC_FAIL2 = 2
 
 
-def exec_cmd(cmd, env=None, stdin=None):
+def exec_cmd(cmd, env=None, stdin=None, check=False):
```

Comment:
The commit message needs to be adjusted.

Reply:
Opps, done. Thanks!